frontend/liferay-theme/src/main/webapp/templates/portal_normal.ftl

```diff
@@ -59,6 +65,15 @@
 <div id="wrapper" class="${hide_portlet_edit_decorators_css} ${login_css}">
 	<header id="banner" role="banner">
 		<div id="heading" class="container">
+            <#if is_signed_in>
+                <#assign expandoAttribute = liferay_user.getExpandoBridge().getAttribute("BannerMessage") />
+                <#if expandoAttribute?has_content>
+                    <div id="updateMessage">
+                        <span class="closebtn" onclick="clickCounter()">&times;</span>
```

Comment:
please rename the function `clickCounter()` to more meaningful (based on action being performed on click of UI element)

Reply:
my bad, forgot to change it to closeBanner.